### PR TITLE
Added project base header 

### PR DESCRIPTION
**Pull request by vanyavidivici**
- Added header component
- Upload necessary images to assets folder
- Insert header component into profile page
- Added flowbite plugin into project

### CODE REVIEW

**Comment by vanyavidivici** [comment]
Also added flowbite plugin into project

**Review by ikew007 — APPROVED** [review]
Great

**Review by NightStrang6r — APPROVED** [review]
Imba

File: playPrism-web/src/modules/user/components/profile/profile.component.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<app-header/>
```
Comment (NureDzabakhidzeDavyd) [inline playPrism-web/src/modules/user/components/profile/profile.component.html:1]: Please, pass app-header in app.component.html file and  import SharedModule to app component

image

Comment (NureDzabakhidzeDavyd) [inline playPrism-web/src/modules/user/components/profile/profile.component.html:1]: Please, pass app-header in app.component.html file and import SharedModule to app component

image

File: playPrism-web/src/modules/shared/components/header/header.component.css.map [hunk 2]
```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":[],"names":[],"mappings":"","file":"header.component.css"}
```
Comment (NureDzabakhidzeDavyd) [inline playPrism-web/src/modules/shared/components/header/header.component.css.map]: Remove css files if they are unneccessary

**Comment by NureDzabakhidzeDavyd** [comment]
> Imba

A Eto Gelik

File: playPrism-web/src/modules/shared/components/header/header.component.css.map (hunk 2 above)
Comment (vanyavidivici) [reply playPrism-web/src/modules/shared/components/header/header.component.css.map]: already fixed

**Pull request merged**